x-pack/legacy/plugins/siem/server/lib/events/query.events_over_time.dsl.ts

```diff
@@ -8,7 +8,7 @@ import { RequestBasicOptions } from '../framework';
 
 export const buildEventsOverTimeQuery = ({
   filterQuery,
-  timerange: { from, to },
+  timerange: { from, time_zone, to },
```

Comment:
Since `time_zone` is optional form the graphQl type, do we need to be more careful here.